### DISCUSSION TITLE
Okta or Github Enterprise Authentication in my VS code extension

### DISCUSSION THREAD

**Discussion by abhishek-sultaniyaTS** [microsoft/vscode-discussions#1238] [Extension Development QnA]
Hi I have made a extension. I want before user uses that extension they need to authenticate with their identity. I want a github enterprise authentication but unable to do so.
My organisation uses OKTA. It will also be fine. If someone has done any of these please guide.
Help would be highly appreciated.

**Comment by isidorn** [comment]
This should just work. What errors are you seeing when trying this?
Use the Authetnication API

**Comment by abrilurena** [comment]
Hi, @abhishek-sultaniyaTS! I hope you're doing great. It's been a while since last time we have heard from you, do you still need help with this?

**Comment by burkeholland** [comment]
Hey, @abhishek-sultaniyaTS - 2nd check for an update here. If you have one, please update this discussion. If this item is still open during our next moderation pass, we'll close it. You can always re-open if it's still an issue. Thanks!

**Comment by olguzzar** [comment]
Closing this discussion as it's been a while without any updates - please feel free to reopen if you have new information!
